ArduinoCode/ArduinoCode.ino

```diff
@@ -271,7 +293,14 @@ double GripperFeedback() {
 }
 void OutputRotate() {
   //rotates stepper number of steps found in feedback
-  rotateStepper.step(stepsToRotate);
+  if (RotateMode == VELOCITY) {
+    if (RotateSetpoint != 0) {
+      //included counting steps here due to problem of missed steps. This seems to have fixed it. Further testing required.
+      rotateStepper.step(rotateDir * 10);
+      rotateCurrentSteps = rotateCurrentSteps + rotateDir * 10;
+      RotateCurrent = rotateCurrentSteps / stepsPerRevolution * 360;
```

Comment:
this answers my previous question but why is this in the output and not the feedback?